hw/top_darjeeling/data/top_darjeeling.hjson

```diff
@@ -73,7 +73,6 @@
     // src:  From which clock source is the clock derived
     // div:  Ratio between derived clock and source clock
     derived_srcs: [
-      { name: "io_div4", aon: "no", div: 4, src: "io", freq: "250000000" }
```

Comment:
Similarly here, `clk_io_div4` is retained in some of Darjeeling's testplans and config files.